lib/jit_preloader/active_record/associations/preloader/ar7_branch.rb

```diff
@@ -0,0 +1,24 @@
+module JitPreloader
+  module PreloaderBranch
+    """
+    From ActiveRecord version >= 7.x.x, it introduces a improvement for preloading associations in batch
```

Comment:
@fimmtiu If you can help double check these comments, I will greatly appreciate it.

Reply:
Sure, I'd be happy to take a stab at rewording it. What do you think?

Reply:
Looks good to me! 🚀